src/main/java/org/worldcubeassociation/db/Persons.java

```diff
@@ -38,4 +38,14 @@ public Person findById(String aId) {
         return existingPersons;
     }
 
+    public boolean existNameAndCountry(String aName, String aCountry) {
```

Comment:
Oh, neat! If the method is unused, please change it to use `.equals`. I don't think efficiency is a concern here.